README.md

```diff
@@ -751,6 +751,10 @@ Table of Contents
    * [webhookrelay.com](https://webhookrelay.com) — Manage, debug, fan-out and proxy all your webhooks to public or internal (ie: localhost) destinations. Also, expose servers running in a private network over a tunnel by getting a public HTTP endpoint (`https://yoursubdomain.webrelay.io <----> http://localhost:8080`).
    * [ZeroTier](https://www.zerotier.com) — FOSS managed virtual Ethernet as a service. Unlimited end-to-end encrypted networks of 100 clients on free plan. Clients for desktop/mobile/NA; web interface for configuration of custom routing rules and approval of new client nodes on private networks.
 
+## Virtual Private Network (VPNs)
+
+   * [tachyon.eco](https://tachyon.eco/) - The #1 blockchain VPN driven by IPX Token, Free And Unlimited, No need to worry about privacy leaks when you connect to    Starbucks or other public WiFi, P2P network without central nodes or companies Multi-path routing to prevent single-point attacks Take control of your own privacy. Never miss videos & messages when you access to a restricted network like school WiFi, 9 tunnels to help you bypass filters and firewalls Protocol simulation to disguise traffic in transit Free access to Facebook, Netflix, Spotify and other platforms. Say goodbye to disconnection to Fortnite during fighting, 90% connection rate in weak, censored networks 95% stability in adversarial environments 200%~1000% faster than other solutions.
```

Comment:
Sorry, I dont think this qualifies for the list.

Apart from that, we dont add single entry catagories and this description is way too long.